src/views/Pools/components/BountyModal.tsx

```diff
@@ -28,23 +25,32 @@ const Divider = styled.div`
   width: 100%;
 `
 
-const BountyModal: React.FC<BountyModalProps> = ({
-  cakeBountyToDisplay,
-  dollarBountyToDisplay,
-  totalPendingCakeHarvest,
-  callFee,
-  onDismiss,
-  TooltipComponent,
-}) => {
+const BountyModal: React.FC<BountyModalProps> = ({ onDismiss, TooltipComponent }) => {
   const { t } = useTranslation()
   const { account } = useWeb3React()
   const { theme } = useTheme()
   const { toastError, toastSuccess } = useToast()
   const cakeVaultContract = useCakeVaultContract()
-  const [pendingTx, setPendingTx] = useState(false)
+  const [pendingTx, setPendingTx] = useState(true)
+  const {
+    estimatedCakeBountyReward,
+    totalPendingCakeHarvest,
+    fees: { callFee },
+  } = useCakeVault()
+  const cakePriceBusd = usePriceCakeBusd()
   const callFeeAsDecimal = callFee / 100
   const totalYieldToDisplay = getBalanceNumber(totalPendingCakeHarvest, 18)
-  const { targetRef, tooltip, tooltipVisible } = useTooltip(<TooltipComponent />, {
+
+  const estimatedDollarBountyReward = useMemo(() => {
+    return new BigNumber(estimatedCakeBountyReward).multipliedBy(cakePriceBusd)
+  }, [cakePriceBusd, estimatedCakeBountyReward])
+
+  const hasFetchedDollarBounty = estimatedDollarBountyReward.gte(0)
+  const hasFetchedCakeBounty = estimatedCakeBountyReward ? estimatedCakeBountyReward.gte(0) : false
+  const dollarBountyToDisplay = hasFetchedDollarBounty ? getBalanceNumber(estimatedDollarBountyReward, 18) : 0
+  const cakeBountyToDisplay = hasFetchedCakeBounty ? getBalanceNumber(estimatedCakeBountyReward, 18) : 0
+
+  const { targetRef, tooltip, tooltipVisible } = useTooltip(<TooltipComponent fee={1000} />, {
```

Comment:
`fee={1000}` should probably be using `callFee`?

Reply:
Sorry it was a left over from testing, fixed now :)